### DISCUSSION TITLE
俺找到了mosdns v3的文档了, 不谢

### DISCUSSION THREAD

**Discussion by Tmalll** [IrineSistiana/mosdns#611] [General]
这个应该是改版前吧,对应3.90
https://web.archive.org/web/20220531073515/https://github.com/IrineSistiana/mosdns

这个可以看其他时期的镜像
https://web.archive.org/web/20220801000000*/https://github.com/IrineSistiana/mosdns

**Comment by w22gb8** [comment]
不错，还在用v3蛮稳的，懒得折腾了。
